### PR TITLE
Fix typos in www/

### PR DESCRIPTION
**Pull request by szepeviktor**
Found some misspellings with `typos`.
Excluded paths:
- dist/
- test/lib/
- test/realtime/static/data.json
- www/static/test/

Please regenerate `dist` files after this PR. I am totally lost in this repo!

### CODE REVIEW

**Comment by szepeviktor** [comment]
Should I revert `test/lib/` files?

**Comment by alexpetros** [comment]
Hi @szepeviktor, thanks for contributing! Would you please remove the changes that are *not* in the `www` folder? Per the contribution guidelines, non-docs changes are pulled against the `dev` branch, not the `main` branch.

**Comment by szepeviktor** [comment]
All right ...

**Comment by szepeviktor** [comment]
Done 🍏

File: www/content/docs.md [hunk 1]
```diff
@@ -1340,9 +1340,9 @@ In hyperscript you can implement similar functionality like so:
 
 ```html
 <div hx-target="#content" _="on htmx:beforeOnLoad take .active from .tabs for event.target">
-    <a class="tabs active" hx-get="/tabl1" >Tab 1</a>
-    <a class="tabs" hx-get="/tabl2">Tab 2</a>
-    <a class="tabs" hx-get="/tabl3">Tab 3</a>
+    <a class="tabs active" hx-get="/tab1" >Tab 1</a>
+    <a class="tabs" hx-get="/tab2">Tab 2</a>
+    <a class="tabs" hx-get="/tab3">Tab 3</a>
```
Comment (alexpetros) [inline www/content/docs.md:1345]: these are not typos (as far as I can tell)

**Review by alexpetros — CHANGES_REQUESTED** [review]
First of all, thank you for removing the non-docs changes. Only one bit of feedback, and with that change it otherwise looks great.

File: www/content/docs.md (hunk 1 above)
Comment (szepeviktor) [reply www/content/docs.md:1345]: Yes, I know.
`typos`˙- the tool I used found it.
Should they be `table1` or `tab-left1`?

Comment (szepeviktor) [reply www/content/docs.md:1345]: Please see the original example in Intercooler.js https://intercoolerjs.org/attributes/ic-switch-class.html

Comment (szepeviktor) [reply www/content/docs.md:1345]: All right, reverting.
Note: "tabl" is not a word.

**Review by alexpetros — APPROVED** [review]
thanks for making those changes!

**Comment by szepeviktor** [comment]
You're welcome.

My heart aches 🙃 tabl, tabl, tabl ...

**Pull request merged**